### PR TITLE
support darwin in prow/bump.sh, ship it

### PR DESCRIPTION
**Pull request by BenTheElder**
This should pick up the recent improvements to deck in particular

waiting to run `bazel run //prow/cluster:production.apply --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64` next 🤞

### CODE REVIEW

File: prow/bump.sh [hunk 1]
```diff
@@ -18,19 +18,19 @@ set -o errexit
 # See https://misc.flogisoft.com/bash/tip_colors_and_formatting
 
 color-image() {  # Bold blue
-  echo -e "\e[1;35m${@}\e[0m"
+  echo -e "\x1B[1;35m${@}\x1B[0m"
```
Comment (BenTheElder) [inline prow/bump.sh]: this is hex for the esc char instead of the \e variant which darwin's ancient bash doesn't support
cc @ixdy @stevekuznetsov :joy:

Comment (stevekuznetsov) [reply prow/bump.sh]: This is why I suggested to use `tput` in the original PR :D

Comment (BenTheElder) [reply prow/bump.sh]: FWIW I did originally do the push from linux, but the gcr push is borked on that machine :( @krzyzacy

**Comment by BenTheElder** [comment]
@qhuynh96 your cookie config is invalid 😱 :(

**Pull request merged**